### PR TITLE
Rebrand our docker image under the hyperledger name

### PR DESCRIPTION
**Pull request by ghaskins**
This is a rebase of #1402 to fix a merge conflict with mainline.  Please see #1402 for description/discussion

Signed-off-by: Greg Haskins gregory.haskins@gmail.com

### CODE REVIEW

**Comment by srderson** [comment]
It looks like this requires users to run `vagrant destroy` `vagrant up`? I will accept this and https://github.com/hyperledger/fabric/pull/1264 together and post a notice to the #fabric-dev channel.

**Comment by ghaskins** [comment]
@srderson yes, destroy/up cycle will be necessary.  I agree on the bundling.

**Comment by srderson** [comment]
LGTM

File: gossip/privdata/coordinator_test.go [hunk 1]
```diff
@@ -1174,18 +1163,20 @@ func TestCoordinatorStoreBlock(t *testing.T) {
...
 	fmt.Println("Scenario IV")
 	// Scenario IV: Block came with more than sufficient private data alongside it, some of it is redundant.
 	pvtData = pdFactory.addRWSet().addNSRWSet("ns1", "c1", "c2", "c3").
 		addRWSet().addNSRWSet("ns2", "c1", "c3").addRWSet().addNSRWSet("ns1", "c4").create()
 	err = coordinator.StoreBlock(block, pvtData)
-	assertPurged("tx1", "tx2")
 	assert.NoError(t, err)
 	assertCommitHappened()
+	purgedTxns = []string{"tx1", "tx2"}
+	require.Eventually(t, assertPurgedTxns, 2*time.Second, 100*time.Millisecond)
```
Comment (manish-sethi) [inline gossip/privdata/coordinator_test.go]: I find this change reducing the readability by isolating the parameters from the function. I feel that a better change would be `func() bool { return assertPurgedTxns("tx1", "tx2") }` (as you have done for the other function below).

**Pull request merged**